hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/QuorumJournalManager.java

```diff
@@ -250,6 +262,9 @@ Map<AsyncLogger, NewEpochResponseProto> createNewUniqueEpoch()
   
   @Override
   public void format(NamespaceInfo nsInfo, boolean force) throws IOException {
+    if (isEnableJnMaintenance()) {
+      throw new IOException("format() does not support enabling jn maintenance mode");
```

Comment:
nit:
`isEnableJnMaintenance(`) -> `isJNMainteanceEnabled()` or should it actually be `IsJNInMaintenanceMode()` ?

"format() does not support enabling jn maintenance mode" -> "Formatting a journal node is not supported while in JN maintenance mode"